global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+        "version": "7.0.201"
```

Comment:
I see they advertise 7.0.203 as the latest 7.0.2xx security update on the official page :) https://dotnet.microsoft.com/en-us/download/dotnet/7.0

Is there a reason for sticking to 201? :)

Reply:
Yes, it seems the latest devcontainer available is still at 201. The SDK version is only used during builds and the runtime version used is the latest one.